### PR TITLE
Add CONTRIBUTING.MD sentence

### PR DESCRIPTION
**Pull request by erik-whiting**
Addresses https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2312

@matentzn, this felt like the most appropriate place on the collaboration page to add the sentence though it does feel out of place to me. Do you think it should be somewhere else or is it fine where it is?

cc @cthoyt

### CODE REVIEW

**Comment by cthoyt** [comment]
The diff on this pr is weird, why are so many sentences on a single line? Can you isolate the addition to its own line so it’s possible to actually see what changed?

**Comment by erik-whiting** [comment]
the change is just the last sentence in the highlighted paragraph

**Comment by cthoyt** [comment]
To clarify, this entire paragraph is one one line, and therefore the entire paragraph gets highlighted if anything is added to this line. Can you simply add a new line before the changes you made?

**Comment by matentzn** [comment]
@cthoyt i added the line break, looks good.

**Review by matentzn — CHANGES_REQUESTED** [review]
This requires a @nataled review so I will undo my approval; but I approve.

File: principles/fp-010-collaboration.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ The benefits of collaboration are threefold: (1) Avoid duplication of work; (2)
...
+Additionally, provide a `CONTRIBUTION.md` file to inform community members how to contribute to your projects ([example](https://github.com/mapping-commons/sssom/blob/master/CONTRIBUTING.md), [learn more](http://mozillascience.github.io/working-open-workshop/contributing/)).
```
Comment (cthoyt) [inline principles/fp-010-collaboration.md:21]: Can we give an example of an ontology's contribution guidelines instead of other kinds of projects?

Comment (cthoyt) [reply principles/fp-010-collaboration.md:21]: Oop I didn't see nico already commented on this issue

**Comment by nataled** [comment]
There are three possible principles this could be added to: 
- Principle 8 (Documentation)
- Principle 10 (Collaboration)
- Principle 20 (Responsiveness)

I personally favor P20 since that's the principle that requires a tracker and this document appears to refer to tracking. @cstoeckert @hoganwr what is your opinion? 

As an aside: no change to a principles page should be made without prior consultation with the EWG (preferably prior to the point of a PR), as it is that group that is charged with overseeing the content of these pages.

**Comment by matentzn** [comment]
> As an aside: no change to a principles page should be made without prior consultation with the EWG (preferably prior to the point of a PR), as it is that group that is charged with overseeing the content of these pages.

100%. @erik-whiting can you make sure this is reflected in our website SOP?

I personally favour the collaboration principle for this change, because the contribution guide basically provides a guide on _how_ to collaborate with a project - which implicitly signals collaboration willingness.

**Comment by nataled** [comment]
There's a distinction we make between collaborators and users-that-make-contributions (this is a very old topic of discussion, dating to the earliest days of the EWG). I prefer 'responsiveness' because that's the principle that governs the interplay between ontology developers (aka collaborators) and the users. One can make the argument that this contributions guidelines document pertains to collaborators, but in my mind that's far less of a need than one that pertains to user contributions since collaborators would (mostly) already know the procedures. That being said, there is the possibility of having it mentioned in multiple places if it makes sense to do so. Here's the question that to my mind tells where this should go:

Is the contributions guideline intended to govern internal interactions (between collaborators) or external interactions (between developers and users)?

**Comment by cstoeckert** [comment]
My opinion is that this belongs in Principle 8 (Documentation). From [Wrangling Web Contributions: How to Build a CONTRIBUTING.md](http://mozillascience.github.io/working-open-workshop/contributing/),  "A CONTRIBUTING.md file ... provides potential project contributors with a short guide to how they can help ..." Sounds like documentation to me and could go on the [Principle: Documentation page](https://obofoundry.org/principles/fp-008-documented.html) under Examples in the developer documentation section.

**Comment by matentzn** [comment]
The truth is, as @nataled has already stated, it belongs in all three:

CONTRIBUTING.md

1. documents the collaborative workflows (documentation)
2. facilitates collaboration by providing a set of rules and regulations (collaboration)
3. codifies the modus operandi for interactions of the user community with the developer team (responsiveness)

I don't want to make this a major debate - I will withdraw my suggestion so you can agree on where to put by yourselves - or just add a note to all three principles!

**Review by cthoyt — APPROVED** [review]

File: principles/fp-010-collaboration.md [hunk 2]
```diff
@@ -26,14 +26,15 @@ It is expected that Foundry ontologies will collaborate with other Foundry ontol
 - _Contribution to external ontology_: Plant Ontology (PO) curators contribute definitions to Gene Ontology (GO) for biological processes and cell components in plants. PO then uses the GO terms in their definition of corresponding structures and stages.
 - _Documentation of collaboration_: Cell Line Ontology (CLO), Cell Ontology (CL), and Ontology of Biomedical Investigations (OBI) published a paper sorting out their overlap and documented working together. Sarntivjiai et al., "CLO: The cell line ontology", J. Biomed. Sem., 2014, 5, 37. http://www.ncbi.nlm.nih.gov/pubmed/25852852
 - _Providing terms upon request_: The Disease Ontology (DO) responded to a request from the PRotein Ontology for curation of certain disease terms.
+- _Providing guidance on how to contribute_: The Relations Ontology (RO) GitHub includes a [CONTRIBUTIONS.md](https://github.com/oborel/obo-relations/blob/master/CONTRIBUTING.md) file.
```
Comment (matentzn) [inline principles/fp-010-collaboration.md:29]:
```suggestion
- _Providing guidance on how to contribute_: The Relations Ontology (RO) GitHub includes a [CONTRIBUTING.md](https://github.com/oborel/obo-relations/blob/master/CONTRIBUTING.md) file.
```

File: principles/fp-010-collaboration.md [hunk 3]
```diff
@@ -26,14 +26,15 @@ It is expected that Foundry ontologies will collaborate with other Foundry ontol
...
+Ontology developers must document their attempts at an open dialog with OBO Foundry members, for example by attempting to ascertain if there are other possible ontologies in (or overlapping) the domain of interest. Such documentation can be a simple pointer to an e-mail thread on the OBO discuss list. If there are other ontologies that might need to be aligned or have boundaries determined, evidence of coordination or cooperation should be provided. Further evidence of collaboration may include examples of terms that have been provided to other ontologies in the OBO Foundry community, while evidence for a willingness to collaborate includes the creation of a CONTRIBUTIONS.md file to provide guidance for contributions ([learn more](http://mozillascience.github.io/working-open-workshop/contributing/)). Finally, hosting or participating in collaborative workshops or meetings attended by OBO Foundry community members is considered evidence of collaboration.
```
Comment (matentzn) [inline principles/fp-010-collaboration.md:37]:
```suggestion
Ontology developers must document their attempts at an open dialog with OBO Foundry members, for example by attempting to ascertain if there are other possible ontologies in (or overlapping) the domain of interest. Such documentation can be a simple pointer to an e-mail thread on the OBO discuss list. If there are other ontologies that might need to be aligned or have boundaries determined, evidence of coordination or cooperation should be provided. Further evidence of collaboration may include examples of terms that have been provided to other ontologies in the OBO Foundry community, while evidence for a willingness to collaborate includes the creation of a CONTRIBUTING.md file to provide guidance for contributions ([learn more](http://mozillascience.github.io/working-open-workshop/contributing/)). Finally, hosting or participating in collaborative workshops or meetings attended by OBO Foundry community members is considered evidence of collaboration.
```

File: principles/fp-010-collaboration.md (hunk 2 above)
Comment (nataled) [reply principles/fp-010-collaboration.md:29]: Oops, sorry, I forgot the original text had that typo and just cut and pasted. Thanks for catching it!

**Comment by nataled** [comment]
@matentzn can you retract your changes? Then this can be merged.

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
You mean "merge" the changes, right? I did that! CONTRIBUTING.md instead of CONTRIBUTION.md

**Comment by nataled** [comment]
@matentzn sorry, I was referring to the original sentence that was inserted, which I moved.

**Comment by nataled** [comment]
I checked, all looks fine, so I completed the process.

**Pull request merged**